Core/CMP API Specification.md

```diff
@@ -564,7 +564,7 @@ gppVersion : Number, // The version number parsed from the header
 
 sectionList : Array of Number, // the sections contained within the encoded GPP string as parsed from the header
 
-applicableSection: Array of Number, // Section ID considered to be in force for this transaction. The value should be 0 during stub/load. In most cases, this field should have a single section ID. In rare occasions where such a single section ID can not be determined, the field may contain up to 2 values. When no section is applicable, the value will be -1.
+applicableSection: Array of Number, // Section ID considered to be in force for this transaction. In most cases, this field should have a single section ID. In rare occasions where such a single section ID can not be determined, the field may contain up to 2 values. The value can be 0 or a Section ID specified by the Publisher / Advertiser, during stub / load. When no section is applicable, the value will be -1.
```

Comment:
applicableSections

Reply:
Fixed

Reply:
I still think you need to fix the max length of two problem